pandas/tests/io/parser/test_quoting.py

```diff
@@ -66,7 +66,6 @@ def test_quote_char_basic(all_parsers):
     tm.assert_frame_equal(result, expected)
 
 
-@xfail_pyarrow  # ValueError: The 'quoting' option is not supported
```

Comment:
I’m pretty sure there are a bunch of these. I’ll try to find my old branch when back at my desk

Reply:
looks like i was thinking of "quoting", not "quotechar", never mind.